### PR TITLE
chore(RHTAPWATCH-686): handle runner exceptions

### PR DESCRIPTION
**Pull request by avi-biton**
Except exceptions from runner
Add errors to the output
Add unit tests

### CODE REVIEW

File: tests/test_rpm_verifier.py [hunk 1]
```diff
@@ -80,6 +81,66 @@
...
+                ),
+                ProcessedImage(image="i2", unsigned_rpms=["their-rpm"]),
+            ],
+            True,
+            dedent(
+                """
+                Found unsigned RPMs:
+                image: i2 | unsigned RPMs: their-rpm
+                Errors encountered:
+                i1: Failed to run command
```
Comment (yftacherzog) [inline tests/test_rpm_verifier.py:97]: I think we should make the format similar between the two failure reasons. Something like:
```suggestion
                Found unsigned RPMs:
                i2 | their-rpm
                Encountered errors:
                i1: Failed to run command 
```

**Review by yftacherzog — COMMENTED** [review]
I think the format between the two kind of failures should be similar. Otherwise, lgtm.

File: tests/test_rpm_verifier.py (hunk 1 above)
Comment (avi-biton) [reply tests/test_rpm_verifier.py]: Done, PTAL

**Review by yftacherzog — APPROVED** [review]
lgtm

**Comment by ifireball** [comment]
@yftacherzog If you're not seeing the pipeline running in RHTAP, perhaps this PR is not based on a version of the code that contains the pipeline

**Pull request merged**